crates/ruff_linter/src/rules/pylint/rules/too_many_statements.rs

```diff
@@ -90,6 +90,7 @@ fn num_statements(stmts: &[Stmt]) -> usize {
             Stmt::Match(ast::StmtMatch { cases, .. }) => {
                 count += 1;
                 for case in cases {
+                    count += 1;
                     count += num_statements(&case.body);
                 }
```

Comment:
I'm unsure of whether `case` should be counted here but looking at the logic for other statements, I think it should.

Reply:
Hm, feel like we should count it yeah.

Reply:
Yeah, I was just thinking out loud :)